src/Form/_Form.scss

```diff
@@ -242,21 +242,22 @@ $select-icon-padding: .5625rem !default;
       opacity: 0;
     }
   }
-  select.form-control {
-    padding-right: 2rem + $select-icon-padding;
-    background-image: $custom-select-indicator;
-    background-position: right $select-icon-padding center;
-    background-repeat: no-repeat;
-    background-size: 24px 24px;
-    appearance: none;
-  }
   select.form-control:not(.has-value):not(:focus) {
     // color: rgba(0,0,0,0); Force the rgba syntax to appear in the output rather
     // than transparent. IE11 does not understand color: transparent here.
     color: unquote("rgba(0,0,0,0)");
   }
 }
 
+select.form-control {
```

Comment:
this is done to fix arrow for selects without floating label as it is currently a bit off
![image](https://user-images.githubusercontent.com/52399399/150085597-ed03421b-7e91-4247-a4bf-3a53b9000a09.png)
this was causing issues for theme switcher select in the header

this change adjusts arrow to be as follows
![image](https://user-images.githubusercontent.com/52399399/150085857-04214005-9ffb-4dea-97a0-806dc2d90aab.png)
which is how it looks for selects with floating label

see https://openedx.github.io/paragon/components/form/ for examples

Reply:
Great catch! 🙌